apps/rosen/app/_networks/cardano/index.ts

```diff
@@ -27,7 +47,13 @@ import { CardanoIcon } from '@rosen-bridge/icons';
 const CardanoNetwork: Network<Wallet> = {
   name: Networks.cardano,
   label: 'Cardano',
-  supportedWallets: [namiWalletInfo],
+  supportedWallets: [
+    namiWalletInfo,
+    laceWalletInfo,
+    eternlWalletInfo,
+    flintWalletInfo,
+    vesprWalletInfo,
```

Comment:
It seems Vespr is a mobile-only wallet, which is not supported in current implementation of Rosen app. Please remove this line. There is no need to remove other logic related to Vespr, though, as we may support it later.